maxim/models/dataset.py

```
@@ -24,7 +24,10 @@ def dict_to_class(cls, data: Dict[str, Any]) -> "DatasetRow":
 
 
 class Variable:
-    def __init__(self, type: str, payload: Dict[str, Union[str, int, bool]]):
+
+    def __init__(
```

Ruff (0.11.9)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)